app/views/foods/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Add new food here</h1>
+<%= form_with mode: @new_food, url: foods_path do |form| %>
```

Comment:
Please display the error message here when the form submission is failed. you can use the code below:
`  <%if @new_food.errors.any?%>

    <div class='alert alert-danger'>
      <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Danger:"><use xlink:href="#exclamation-triangle-fill"/></svg>

    <% @recipe.errors.full_messages.each do |msg|%>
      <p><%=msg%></p>
     <%end%>
    </div>
  <%end%>`